scripts/get_metrics

```diff
@@ -223,12 +223,13 @@ def process_storage(storage, args):
     if args.output is not None:
         columns = [Timestamp()]
         for m in MetricsName:
-            columns.append(MetricsValue(m, 'sum'))
-            columns.append(MetricsValue(m, 'count'))
-            columns.append(MetricsValue(m, 'min'))
-            columns.append(MetricsValue(m, 'lo'))
-            columns.append(MetricsValue(m, 'avg'))
-            columns.append(MetricsValue(m, 'hi'))
+            if m < 30000:
```

Comment:
Why?

Reply:
But further it would be nice to have something like _temp_seed = 30000_ and condition _if m < temp_seed:_, but merged due to urgency.